packages/common/src/search/fetch-datasets.ts

```diff
@@ -0,0 +1,18 @@
+import { DatasetResource, IHubRequestOptions, hubApiRequest } from "..";
+
+export function fetchDatasets(
+  requestOptions: IHubRequestOptions
```

Comment:
As per [this comment](https://github.com/Esri/hub.js/issues/606#issuecomment-912107464), don't we need to have this function take in an `IContentSearchRequest`? @andrewctate what would be the ramifications if we left it this way?

Reply:
see https://github.com/Esri/hub.js/pull/643#issuecomment-926008791, that's coming in the next PR

Reply:
Sorry, I didn't distinguish between `searchDatasets` and `fetchDatasets` 😅  You're good to go!